### PR TITLE
Fix/after migration

### PR DESCRIPTION
**Pull request by NadiaKishchuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/features/AdminPage/Login/AdminLogin.style.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+@use "@sass/mixins/_utils.mixins.scss" as mut;
+@use "@sass/variables/_variables.fonts.scss" as ft;
+@use "@sass/variables/_variables.colors.scss" as c;
+@use "@sass/_utils.functions.scss" as f;
+.admin-login-form{
+    @include mut.sized(400px,200px);
+    margin: auto;
+
```
Comment (Tysyatsky) [inline src/features/AdminPage/Login/AdminLogin.style.scss:9]: remove spacing

File: src/app/router/Routes.tsx [hunk 2]
```diff
@@ -18,18 +20,38 @@ import StreetcodeCatalog from '@/features/StreetcodeCatalogPage/StreetcodeCatalo
...
+            path={FRONTEND_ROUTES.ADMIN.DICTIONARY}
+            element={(
+                <ProtectedComponent>
+                    <TermDictionary />
+                </ProtectedComponent>
+            )}
+        />
+        <Route index path={FRONTEND_ROUTES.CATALOG.BASE} element={<StreetcodeCatalog />} />
+        <Route index path={FRONTEND_ROUTES.ADMIN.LOGIN} element={<AdminLogin />} />
+        {/* <Route path='*' element={<NotFound />} /> */}
```
Comment (Tysyatsky) [inline src/app/router/Routes.tsx:54]: commented line

**Review by Tysyatsky — APPROVED** [review]
Great work! Fix 2 comments and merge it freely.

File: src/app/router/Routes.tsx (hunk 2 above)
Comment (NadiaKishchuk) [reply src/app/router/Routes.tsx:54]: Honestly, I don`t know why it is commented on, it wasn`t done by me.

**Pull request closed without merging**